source/_integrations/smlight.markdown

```diff
@@ -45,3 +46,10 @@ The following sensors will be created:
 - **Zigbee Uptime** - Uptime of Zigbee connection to ZHA/Z2M
 - **RAM Usage** - Monitor RAM Usage
 - **FS Usage** - Monitor filesystem usage
+
+The following buttons will be created:
+
+- **Core Restart** - Restart core ESP32
+- **Zigbee Restart** - Restart Zigbee CCCC2652 or EFR32 chip
+- **Zigbee Flash Mode** - Trigger the Zigbee chip into bootloader flash mode so it can be flashed. It is possible to flash Zigbee firmware over a network socket once this is activated.
```

Comment:
```suggestion
- **Core restart** - Restart core ESP32
- **Zigbee restart** - Restart Zigbee CCCC2652 or EFR32 chip
- **Zigbee flash mode** - Trigger the Zigbee chip into bootloader flash mode so it can be flashed. It is possible to flash Zigbee firmware over a network socket once this is activated.
```
I realize the other UI strings here are capitalized. But we use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization)

Reply:
sure, have applied this change